internal/include/otel_types.h

```diff
@@ -0,0 +1,145 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef _OTEL_TYPES_H
+#define _OTEL_TYPES_H
+
+#include "go_types.h"
+#include "common.h"
+
+/* Defintions should mimic structs defined in go.opentelemetry.io/otel/attribute */
+typedef u64 attr_val_type_t;
+
+#define INVALID 0
+#define BOOL 1
+#define INT64 2
+#define FLOAT64 3
+#define STRING 4
+#define BOOLSLICE 5
+#define INT64SLICE 6
+#define FLOAT64SLICE 7
+#define STRINGSLICE 8
```

Comment:
Shouldn't we inject these into the eBPF space given they are constants we can import in the Go space? This seems highly error prone.

Reply:
I agree, changed